src/components/RoiCalculatorModal/RoiCalculatorFooter.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { Flex, Box, Text, ExpandableLabel, LinkExternal, Grid, HelpIcon, useTooltip } from '@pancakeswap/uikit'
+import { useTranslation } from 'contexts/Localization'
+import { getApy } from 'utils/compoundApyHelpers'
+
+const Footer = styled(Flex)`
+  width: 100%;
+  background: ${({ theme }) => theme.colors.dropdown};
+`
+
+const BulletList = styled.ul`
+  list-style-type: none;
+  margin-top: 16px;
+  padding: 0;
+  li {
+    margin: 0;
+    padding: 0;
+  }
+  li::before {
+    content: '•';
+    margin-right: 4px;
+    color: ${({ theme }) => theme.colors.textSubtle};
+  }
+  li::marker {
+    font-size: 12px;
+  }
+`
+
+interface RoiCalculatorFooterProps {
+  isFarm: boolean
+  apr: number
+  displayApr: string
+  autoCompoundFrequency: number
+  multiplier: string
+  linkLabel: string
+  linkHref: string
+  performanceFee: number
+}
+
+const RoiCalculatorFooter: React.FC<RoiCalculatorFooterProps> = ({
+  isFarm,
+  apr,
+  displayApr,
+  autoCompoundFrequency,
+  multiplier,
+  linkLabel,
+  linkHref,
+  performanceFee,
+}) => {
+  const [isExpanded, setIsExpanded] = useState(false)
+  const { t } = useTranslation()
+  const {
+    targetRef: multiplierRef,
+    tooltip: multiplierTooltip,
+    tooltipVisible: multiplierTooltipVisible,
+  } = useTooltip(
+    <>
+      <Text>
+        The Multiplier represents the proportion of CAKE rewards each farm receives, as a proportion of the CAKE
+        produced each block.
+      </Text>
+      <Text>For example, if a 1x farm received 1 CAKE per block, a 40x farm would receive 40 CAKE per block.</Text>
+      <Text>This amount is already included in all APR calculations for the farm.</Text>
```

Comment:
These need to be translated.

Reply:
Fixed in e39b1877f02e9f195cb159d0f822e0e6617016c5